ykrt/pass_finder/try_passes.py

```diff
@@ -185,18 +211,114 @@ def binary_split(logf, passes, is_prelink):
         print(72 * "=")
         print(list_of_passes_to_str(ok_passes))
 
-def main(logf, is_prelink):
+def evaluate_fitness(glogf, is_prelink, entity, passes, cwd):
```

Comment:
Can we add a short docstring to all newly introduced functions? It will help our future selves.

Reply:
I'd also like that!

Reply:
@vext01 do we want docstring in all of the functions or only the genetic algorithm ones?

Reply:
For now, only new functions you've added in this PR.

If you want to add others, then that should be a separate PR IMHO.

Reply:
Addressed here: afb33c3ac820c5a7164301dfcdc43ea4d1835fd0